test/e2e/chaos/basic.go

```diff
@@ -1799,6 +1771,100 @@ selector:
 
 })
 
+func makeNetworkDelayChaos(
```

Comment:
Should we add a `toLabelSelectors` for this function? or we would set `.Spec.Target` manually (e.g. [this line](https://github.com/chaos-mesh/chaos-mesh/pull/1069/files#diff-6370badef8e5715e4f72fad6d6488b517cd1cca6f98764aa93209e5d3710e36bR1614))

Reply:
OK, I agree.